doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -687,13 +687,126 @@ side. E.g.:
    Notation "'apply_id' f a1 .. an" := (.. (f a1) .. an)
      (at level 10, f ident, a1, an at level 9).
 
+Custom entries
+~~~~~~~~~~~~~~
+
+.. cmd:: Declare Custom Entry @ident
+
+New term and pattern grammar entries, called *custom entries*, can be
+declared using the :n:`Declare Custom Entry @ident` command and later
```

Comment:
I suggest you reformulate as in my previous proposition to avoid repeating this command uselessly.

Reply:
Done.